--- install.sh	2026-09-16 08:46:05.622616554 +0700
+++ install-patch.sh	2026-09-16 08:49:59.514327943 +0700
@@ -184,8 +184,7 @@
 else
   UNAME_MACHINE="$(uname -m)"
 
-  HOMEBREW_PREFIX_DEFAULT="/home/linuxbrew/.linuxbrew"
-  HOMEBREW_CACHE="${HOME}/.cache/Homebrew"
+  HOMEBREW_PREFIX_DEFAULT="${PWD}/home/linuxbrew/.linuxbrew"
 
   STAT_PRINTF=("/usr/bin/stat" "-c")
   PERMISSION_FORMAT="%a"
@@ -821,26 +820,6 @@
 fi
 execute_sudo "${CHOWN[@]}" "-R" "${USER}:${GROUP}" "${HOMEBREW_REPOSITORY}"
 
-if ! [[ -d "${HOMEBREW_CACHE}" ]]
-then
-  execute "${MKDIR[@]}" "${HOMEBREW_CACHE}"
-fi
-if exists_but_not_writable "${HOMEBREW_CACHE}"
-then
-  execute_sudo "${CHMOD[@]}" "g+rwx" "${HOMEBREW_CACHE}"
-fi
-if file_not_owned "${HOMEBREW_CACHE}"
-then
-  execute_sudo "${CHOWN[@]}" "-R" "${USER}" "${HOMEBREW_CACHE}"
-fi
-if file_not_grpowned "${HOMEBREW_CACHE}"
-then
-  execute_sudo "${CHGRP[@]}" "-R" "${GROUP}" "${HOMEBREW_CACHE}"
-fi
-if [[ -d "${HOMEBREW_CACHE}" ]]
-then
-  execute "${TOUCH[@]}" "${HOMEBREW_CACHE}/.cleaned"
-fi
 
 if should_install_command_line_tools
 then
@@ -1032,7 +1011,6 @@
     PATH_WARN=1
   fi
 
-  execute "${HOMEBREW_PREFIX}/bin/brew" "update" "--force" "--quiet"
 
   if [[ -n "${PATH_WARN-}" ]]
   then
